Survey/Sources/Presentation/Modules/Home/HomeView.swift

```diff
@@ -25,24 +25,22 @@ struct HomeView: View {
             text: .constant(""),
             content: { geo in
                 ZStack(alignment: .top) {
-                    ScrollView(.horizontal) {
-                        LazyHStack {
-                            tabViewSetup()
-                                .frame(
-                                    width: geo.size.width,
-                                    height: geo.size.height
-                                )
-                        }
-                        .overlay(alignment: .top) {
-                            HeaderHomeView()
-                                .padding(.top, 60.0)
-                                .padding(.leading, 20.0)
-                        }
+                    LazyHStack {
+                        tabViewSetup(geo: geo)
```

Comment:
tabViewSetup is still redundant with `LazyHStack`, not sure what's the purpose of `LazyHStack` here 🤔